src/corg/coding-and-design-guidelines/code-review-guidelines.md

```diff
@@ -7,74 +7,235 @@ review:
+
+For instance, if their logic for X wasn't so great, but they did Y and Z well, let them know!
+
+#### There's More Than One Solution
+
+Don't always think of how you would have done something. Because it isn't how you would have approached and solved the problem, doesn't mean it isn't good.
+Take into account the initiator's point of view.
+If something is not clear or no consensus is reached, maybe ask the initiator/other reviewers directly before adding a comment.
+Let your ego at the door and be open-minded to considering new solutions.
+Ultimately, pair program before the code review stage!
+
+#### It's not "You", it's "the Code"
+
+Prefer "The logic isn't clear to me." to "Your logic isn't clear."
+
+#### Discuss and Improve the Review Process Often
+
+Ask people what kind of feedback is useful, if they agree with the comments in general, if the comments don't sound too rude, etc.
+Consider how much time people spent doing reviews and identify any patterns from it. If people are frequently commenting on the same types of issues, are there steps we can take to address it so there are less of those issues on the next project? E.g.: improve formatting, add some automatic style checking, have better guidelines, etc.
+
+## After your Pull Request is Reviewed
+
+Read all comments made, and try not to be offended by the short, sharp or abrupt sentence formulations that a reviewer may use. There may not always be complete phrases or "please", or "could you", but that's OK, it's a technical review, not a social encounter.
+
+You must take into account all comments requesting changes, either by doing the requested changes (then it's a good idea to 👍 the comment), or by discussing further and getting to a consensus between you and the commenters.
+
+Reply to any other comment, or simply acknowledge with an appropriate emoji reaction.
+
+If there are some changes to do, there are basically two solutions:
+
+- If the PR contains a few commits, let's say 3 or 4, you should amend the appropriate commits and force push the branch. Don't create further commits for the fixes (again, think about people reading commits one by one to understand the logic of the code).
+- If there are a lot of commits, let's say 5 or more, it's OK and probably a good idea to add some additional commits including the new changes:
+  - It makes things clearer and easier to re-review.
+  - It's safer: rebasing the PR is a risk of impacting some changes that have already been fully reviewed and validated.
+  - It is interesting to have in the history the result of a (possibly long) discussion and process that lead to these additional changes.
+
+Of course, these are only recommendations and there is no absolute way of doing, it always depends on the case.
+
+After you've done fixes and amended code, it's a good idea to add a message in the PR to summarize your changes and ask the reviewers for another review.
```

Comment:
Worth adding to use the `Resolve conversation` button when the commits are amended? Use 👍 for agreeing a request change, use `Resolve conversation` to let know to the reviewer it's done.